test/cli/commands_test.py

```diff
@@ -1256,44 +1256,40 @@ def test_encoding(encoding_in, encoding_out):
         )
 
 
-def test_cli_pass_on_correct_encoding_argument():
```

Comment:
Replaced two tests that were testing the `--encoding` command-line parameter with a single parametrized test which does the same but also tests setting encoding in the `.sqlfluff` config file.